packages/@aws-cdk/cloudformation-diff/test/diff-template.test.ts

```diff
@@ -1104,7 +1104,7 @@ describe('changeset', () => {
             ResourceType: 'AWS::S3::Bucket',
             Replacement: 'True',
             Details: [{
-              Evaluation: 'Direct',
+              Evaluation: 'Static',
```

Comment:
Curious, do you mind explaining why we needed to change the Evaluation type from Direct to Static?

Reply:
This was not a valid value before but in v2 it wasn't strongly typed enough for it to matter in our test case.